yt_dlp/output/formatting.py

```diff
@@ -0,0 +1,144 @@
+import enum
+
+from .hoodoo import Color, TermCode, Typeface, format_text
+from ..utils import format_bytes, timetuple_from_msec, try_call
```

Comment:
`format_bytes` should be implemented in this module since it would fit thematically. `utils` could just wrap this function or reassign.

Reply:
Moved, and maybe less relevant. Those functions could be in `utils` instead if needed by others and then slowly move into sub-modules instead.